### PR TITLE
Fixed container status check

### PR DESCRIPTION
**Pull request by akashkeshari**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/connectedk8s/azext_connectedk8s/custom.py [hunk 1]
```diff
@@ -292,18 +313,21 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name,
...
             print("Exception when calling CoreV1Api->list_namespaced_pod: %s\n" % e)
         for pod in api_response.items:
-            for container_status in pod.status.container_statuses:
-                if container_status.name == 'connect-agent':
-                    connect_agent_state = container_status.state.running
-                    if connect_agent_state is not None:
-                        found_running = found_running + 1
-                        time.sleep(2)
+            if pod.metadata.name.startswith('connect-agent'):
+                for container_status in pod.status.container_statuses:
```
Comment (rapatchi) [inline src/connectedk8s/azext_connectedk8s/custom.py]: Is is possible to merge checking container status and container availability into one step ?

File: src/connectedk8s/azext_connectedk8s/custom.py [hunk 2]
```diff
@@ -292,18 +313,21 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name,
...
+                    if container_status.name == 'connect-agent':
+                        connect_agent_state = container_status.state.running
```
Comment (rapatchi) [inline src/connectedk8s/azext_connectedk8s/custom.py]: could you explain what are the possible values of container_status.state.running ?.

Comment (akashkeshari) [reply src/connectedk8s/azext_connectedk8s/custom.py]: Its is 'None' if the container is not running. If container is running, it contains the start-time of the container

Comment (rapatchi) [reply src/connectedk8s/azext_connectedk8s/custom.py]: okay

**Review by rapatchi — APPROVED** [review]

**Pull request merged**